src/frontend/src/eth/components/send/SendForm.svelte

```diff
@@ -38,7 +38,7 @@
 <form on:submit={() => dispatch('icNext')} method="POST">
 	<div class="stretch">
 		{#if destinationEditable}
-			<SendDestination bind:destination bind:invalidDestination />
+			<SendDestination bind:destination bind:invalidDestination on:qrCodeScan />
```

Comment:
For consistency reason, I suggest to prefix the events with `ic`.

e.g. `icQRCodeScan`

Reply:
on it